addField.sh

```diff
@@ -1,4 +1,10 @@
 #! /bin/bash
+#
+# Add a new field to an existing index.
+# Field mapping is defined in specified file. Must be valid json.
+# Required args: user, env, index, docType, file
```

Comment:
Might be nice to have an example for each script. For example, what is a valid "docType" or "file" based on current setup? Otherwise I play with this every time I am using them.